client/rpcserver/handlers.go

```diff
@@ -431,6 +433,20 @@ func handleWithdraw(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 	return createResponse(withdrawRoute, &res, nil)
 }
 
+func handleRescanWallet(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
+	assetID, err := parseRescanWalletArgs(params)
+	if err != nil {
+		return usage(withdrawRoute, err)
+	}
+	err = s.core.RescanWallet(assetID)
+	if err != nil {
+		errMsg := fmt.Sprintf("unable to rescan wallet: %v", err)
+		resErr := msgjson.NewError(msgjson.RPCWithdrawError, errMsg)
```

Comment:
Need a new `msgjson.RPCRescanError` I guess.

Reply:
Feels odd having a **msgjson** error code that doesn't correspond to any errors returned over the main API, but we're sort of stuck with our previous design choice here in **rpcserver**.

Reply:
Yeah, it's really unfortunate at this point that the same enum is used for both sorts of errors in completely different domains.